roofit/roofit/src/RooNDKeysPdf.cxx

```diff
@@ -240,7 +241,8 @@ RooNDKeysPdf::RooNDKeysPdf(const char *name, const char *title, const RooArgList
    _tracker = new RooChangeTracker("tracker", "track rho parameters", _rhoList, true); // check for value updates
    (void)_tracker->hasChanged(true); // first evaluation always true for new parameters (?)
 
-   createPdf();
+   std::unique_ptr<RooDataSet> ownedData{createDatasetFromHist(varList, hist)};
```

Comment:
Since the data are now not owned by the class but just temporarily created, would not be better to change the name from `ownedData` to just `data` or `tmpData` ?

Reply:
Ah yes that's true! I'll rename it